service_linux.go

```diff
@@ -715,33 +722,55 @@ func plumbProxy(iPort *PortConfig, isDelete bool) error {
 	return nil
 }
 
+func writePortsToFile(ports []*PortConfig) (string, error) {
+	f, err := ioutil.TempFile("", "port_configs")
```

Comment:
We are not removing them but that has been the case forever since this code was there. We can fix it here though.